apps/create/widget/templates/project/default.jsx

```diff
@@ -0,0 +1,217 @@
+let { theme } = props.theme;
+let { variables } = props.variables;
+
+if (!variables) {
+  variables = `
+      --main-color: rgb(68, 152, 224);
+  `;
+}
+
+if (!theme) {
+  theme = `
+
+    [navbar] {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0 20px;
+      height: 60px;
+
+      [left] {
+        display: flex;
+        align-items: center;
+        height: 100%;
+      }
+
+      [logo] {
```

Comment:
what does this do? bracket notation

Reply:
It's the CSS selector for tags attributes like `<div logo></div>` there is no reason I use it, except that like it and it will not collide with gateways CSS and bootstrap